packages/ui-react/src/components/Header/Header.tsx

```diff
@@ -106,7 +111,8 @@ const Header: React.FC<Props> = ({
   });
 
   // only show the language dropdown when there are other languages to choose from
-  const showLanguageSwitcher = supportedLanguages.length > 1;
+  // FEAT:: no language switcher in oauth mode
+  const showLanguageSwitcher = !isOAuthMode && supportedLanguages.length > 1;
```

Comment:
Where supported languages can be defined?

Reply:
in the .env